src/screens/Circuits.js

```diff
@@ -51,8 +47,16 @@ function Circuits () {
                     <div className='mb-4' />
                   </span>
               )))}
-            </div>
-          )
+        </div>
+    )
+  }
+
+  return (
+    <Chrome>
+      {
+        loading
+          ? <Loading />
+          : <CircuitListing circuits={activeCircuits} typeOfCircuits={"Active Circuits"} />
```

Comment:
Technically I renamed this here, before adding Past Circuits.  Shortcut for simplicity sake.